src/DSP.jl

```diff
@@ -1,14 +1,23 @@
 ## DSP.jl ##
 
+## === FUNCTIONS == ##
+
 for (T, suff) in ((Float64, "D"), (Float32, ""))
 
+    """
+    Convolution between an input Vector{T} 'X', and a kernel/filter Vector{T} 'K'.
+
+    Returns: Void. Computation result is stored in 'result'.
```

Comment:
My mistake! This has been changed now; sorry for all these small issues that make reviewing a PR so much more arduous.